packages/openlogin-utils/src/whitelabel.ts

```diff
@@ -0,0 +1,49 @@
+import Color from "color";
+
+import { type WHITE_LABEL_THEME } from "./interfaces";
+
+function getColorsList(colorsAmount = 3, colorsShiftAmount = 50, mixColor = "black", rotate = 0, saturation = 20, mainColor = "#0346ff") {
+  const colorsList = [];
+
+  let step;
+  for (step = 0; step < colorsAmount; step += 1) {
+    colorsList.push(
+      Color(mainColor)
+        .rotate(((step + 1) / colorsAmount) * -rotate)
+        .saturate(((step + 1) / colorsAmount) * (saturation / 100))
+        .mix(Color(mixColor), ((colorsShiftAmount / 100) * (step + 1)) / colorsAmount)
+        .hex()
+    );
+  }
+
+  return colorsList;
+}
+
+function generateWhiteLabelTheme(primary: string) {
```

Comment:
1. Does it make sense to put this function in openlogin-utils? Will it affect build size? If so can we code split it?
2. Should this function instead only return theme colors and we let the caller of the function apply the colors themselves?
Technically the only colors that would need to be returned would be the primary color palette as the other colors don't require other shades being generated by us.

Reply:
1. Yes. it doesn't affect build size.
2. should be fine